drivers/flash/Kconfig.nrf

```diff
@@ -67,6 +67,13 @@ config SOC_FLASH_NRF_PARTIAL_ERASE_MS
 	  Minimal timeout is 2ms maximum should not exceed half of
 	  FLASH_PAGE_ERASE_MAX_TIME_US im ms.
 
+config SOC_FLASH_NRF_TIMEOUT_MULTIPLIER
+    int "Multiplier*10 for flash operation timeouts"
+    default 15
+    help
+      This is a multiplier that will be divided by 10 that is applied to flash
```

Comment:
... to the flash erase or the write operation. Timeout which is base for the multiplication would allows to erase all nRF flash pages in blocking mode.